config/rbac/secrets_role.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openstack-provider-manager-secrets-role
```

Comment:
Please drop redundant `role` from the name
```suggestion
  name: openstack-provider-manager-secrets
```

Reply:
Thought about this as well, but used it because the autogenerated code in rbac_role.yaml uses this notion as well.

But I also think it is nicer without.